examples/README.md

```diff
@@ -1 +0,0 @@
-example is outdated and will probably get removed soon
```

Comment:
why? I found them useful

Reply:
i found them a bit annoying since they are another thing to keep in sync on changes and they have a tendency to get out of sync since there's no tests for it. i'd  rather have the integration tests be the example and refer people to them for examples on config files, node setup, etc